ts/Core/MSPointer.ts

```diff
@@ -19,6 +19,7 @@ const {
     noop,
     win
 } = H;
+import O from './Options.js';
```

Comment:
This should be a type import as well. I will stop pointing these out. Please check all files, if only the type is used. It will help you to move determ the correct file after OptionsLike becomes Options (and Options becomes DefaultOptions).